packages/e2e-tests/src/specs/editor/contributorUser.js

```diff
@@ -37,10 +37,10 @@ describe('Contributor User', () => {
 
     await expect(page).toMatchElement('button', { text: 'Public' });
     await expect(page).not.toMatchElement('li[role="option"]', {
-      text: 'Private',
+      text: 'Private Visible to site admins & editors only',
     });
     await expect(page).not.toMatchElement('li[role="option"]', {
-      text: 'Password Protected',
+      text: 'Password Protected Visible only to those with the password.',
```

Comment:
💭 I think this was good the way it was before! This is much more specific now, which may break the test in ways we don't want.

Ex scenario: The options are visible, but the help text (`Visible to site admins & editors only`) is removed.

Then this test would still fail even though the option is visible. I think it may be better to use a less specific regular expression here:

```js
await expect(page).not.toMatchElement('li[role="option"]', {
  text: /^Private/,
});
await expect(page).not.toMatchElement('li[role="option"]', {
  text: /^Password Protected/,
});
```

Thoughts?

Reply:
hahaha I was just about to update this! Much better.